config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -101,3 +110,146 @@ const (
 	// UserDefinedNamespaceIsolated enables monitoring for user-defined projects with namespace-scoped tenancy. This ensures that metrics, alerts, and monitoring data are isolated at the namespace level.
 	UserDefinedNamespaceIsolated UserDefinedMode = "NamespaceIsolated"
 )
+
+// The MetricsServerConfig resource defines settings for the Metrics Server component.
+// This configuration allows users to customize how the Metrics Server is deployed
+// and how it operates within the cluster.
+type MetricsServerConfig struct {
+	// audit defines the audit configuration used by the Metrics Server instance.
+	// When omitted, this means no opinion and the platform is left to choose a default,
+	// which is subject to change over time. The current default is "metadata".
+	// The audit field is optional.
+	// +optional
+	Audit *Audit `json:"audit,omitempty"`
+
+	// nodeSelector is the node selector applied to metrics server pods.
+	// nodeSelector is optional.
+	//
+	// When omitted, this means the user has no opinion and the platform is left
+	// to choose reasonable defaults. These defaults are subject to change over time.
+	// The current default is `kubernetes.io/os: linux` so that Pods can be scheduled onto any available node.
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// tolerations is a list of tolerations applied to metrics server components
+	// tolerations is optional.
+	//
+	// When omitted, this means the user has no opinion and the platform is left
+	// to choose reasonable defaults. These defaults are subject to change over time.
```

Comment:
Are you actually going to be assigning defaults here?

If we aren't ever going to be configuring defaults for a field, we don't need to put this "no opinion" blurb here.